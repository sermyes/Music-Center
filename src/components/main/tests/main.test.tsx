import Main from '../main';
import '@testing-library/jest-dom';
import renderer from 'react-test-renderer';

describe('Main Component', () => {
  it('renders correctly', () => {
    const component = renderer.create(<Main />);

    expect(component.toJSON()).toMatchSnapshot();
  });
});
