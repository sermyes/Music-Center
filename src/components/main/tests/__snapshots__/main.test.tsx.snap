// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Main Component renders correctly 1`] = `
<div
  className="container"
>
  <div
    className="mainContent"
  >
    <div
      className="titleContainer"
    >
      <h1
        className="title"
      >
        Music Center
      </h1>
      <p
        className="description"
      >
        Free Youtube Music, Melon Top100, Club Music, Show me the money ...
      </p>
    </div>
    <div
      className="contactContainer"
    >
      <a
        className="contact"
        href="mailto:sermyes@gmail.com"
      >
        <i
          className="contact_icon fas fa-envelope"
        />
      </a>
      <a
        className="contact"
        href="tel:010-6807-5339"
      >
        <i
          className="contact_icon fas fa-phone"
        />
      </a>
      <a
        className="contact"
        href="https://github.com/sermyes/musicCenter"
      >
        <i
          className="contact_icon fab fa-github"
        />
      </a>
    </div>
  </div>
  <div
    className="btnContainer"
  >
    <button
      className="btn"
      onClick={[Function]}
    >
      <i
        className="icon fas fa-music"
      />
      Listen Music
    </button>
  </div>
</div>
`;
